### ISSUE TITLE
how to create my first account?

### ISSUE THREAD

**Issue by yousq** [bitshares/bitshares-core#917]
hi,
  I installed a suit of bitshare(a witness node and a cli_wallet daemon) on my ubuntu server,  but I don't exactly know how to create my first account.  

  Both register_account() and create_account_with_brain_key() need a referrer_account, does this mean that it's imposible to create my first account all by myself?

  In addition, what is the difference between register_account(...,broadcast=false) and register_account(...,broadcast=true)?

**Comment by abitmore** [comment]
You can set `referrer_account` to `init0`.

`broadcast=true` means "actually do it", `broadcast=false` means not.

**Comment by abitmore** [comment]
Have you read this? https://github.com/bitshares/bitshares-core/wiki/CLI-Wallet-Cookbook

**Comment by yousq** [comment]
Yeah, I have read that document, 
...
To create a new account, you will need to start with an existing account with some of the CORE asset that will pay the transaction fee registering your new account. The account paying this fee will be the Registrar
...

In my private testnet, I successfully created my accounts by using nathan as the referrer_account, that's because I can import banlance for nathan.

But the problem is that I have no balance(before I have my own accounts) on the main net, who will afford the fee for me to create my first account?

**Comment by abitmore** [comment]
Oh, you're talking about mainnet. First distribution is in genesis.json. Then there will be someone have balance and are lifetime members. They'll create accounts for others. Read the testnet doc, "faucet" part.

**Comment by yousq** [comment]
I have came to get the point, faucet is a web service for users who have no balance to register an account. 
If I set up a chain and a faucet service, others can register an account in my chain whitout any balance via my faucet service.

So if I need to enter the public bts mainnet, I need to access to the official faucet service.
I do not know the where is the  official faucet service, but I can register an account via the UI client on my laptop, and find the private key of it,  and then import the private key on my ubuntu server.

Is that right?
